### ISSUE TITLE
Ubuntu images contain python3

### ISSUE THREAD

**Issue by TeppoLehtonen-TomTom** [adoptium/containers#157]
https://github.com/adoptium/containers/pull/120 added `python-is-python3` package to Ubuntu images. The problem is that [`python-is-python3` depends on `python3`](https://packages.ubuntu.com/focal/python-is-python3) so that gets installed as well.

According to the discussion in https://github.com/adoptium/containers/issues/116, the intent of adding the `python-is-python3` package was not to install `python3`, and that the intent is to have as small diff to bare-bones base image as possible. It seems to me that adding `python3` to the images was unintentional.

The Ubuntu images did not contain `python3` before (as far as I can see), and its inclusion can be seen in the increased sizes of the images, e.g:
https://github.com/docker-library/repo-info/commit/fa669f5fc9650674040499fdd09f787b292667b4#diff-bae803386ba8e4c8e39c393889244a818bc0355b2bac8a0b9bc7dab236dea324
```
REPOSITORY             TAG                    IMAGE ID       CREATED         SIZE
eclipse-temurin        11-jre                 4520f806b3be   30 hours ago    270MB
eclipse-temurin        <none>                 a52f9ce88ddf   2 months ago    270MB
eclipse-temurin        <none>                 dd3751bc18f2   2 months ago    244MB
adoptopenjdk           11-jre                 2c57fb3bc67b   4 months ago    244MB
```

**Comment by gdams** [comment]
Hmm this is a problem, we shouldn't be shipping with python installed. CC @pgerhard - we're going to likely need to pull this out unless there's another approach that prevents the installation of the python3 package

**Comment by pgerhard** [comment]
I will have a look, however as I mentioned in [this comment](https://github.com/adoptium/containers/issues/116#issuecomment-975399517) on #116 Python3 was already installed in the image. The intent of #120 was only to enable the calling of Python3 by using the `python` command in the CLI. If this did change did indeed result in unintentional installation of Python3 I agree that it should be removed.

**Comment by pgerhard** [comment]
So I went back and did some digging.
It appears that the Ubuntu image we are inheriting from does intend not include Python itself. Therefor I created the PR #164 removing the package `python-is-python3`. 

I'm still confused as the why Python is missing in the Ubuntu image, since I distinctly remember it being there back when I suggested adding the package. However since I've moved company since then, I no longer have access to Dockerfile I was working on at the time, so I am unable to investigate further.
